lib/network/encoding.go

```diff
@@ -192,6 +192,7 @@ func MarshalRegisteredType(data ProtocolMessage) ([]byte, error) {
 	var err error
 	if buf, err = protobuf.Encode(data); err != nil {
 		dbg.Error("Error for protobuf encoding:", err)
+		dbg.Error(dbg.Stack())
```

Comment:
I would probably remove it, or only print it if debug level is high (> 3 for instance)